views/apply.njk

```diff
@@ -0,0 +1,35 @@
+{% extends "./layouts/main.njk" %}
+
+{% block pageTitle %}
+  Apply for Role
+{% endblock %}
+
+{% block links %}
+<link rel="stylesheet" href="/css/style.css">
+{% endblock %}
+
+{% block content %}
+
+
+<div class="d-flex justify-content-center "">
```

Comment:
Double quotation marks at the end of the div tag.

Reply:
Removed